suites/squid/cephfs/8_1_features_suite.yaml

```diff
@@ -151,7 +151,7 @@ tests:
   ######################################################################################################################
   # Conf File : conf/squid/cephfs/tier-2_cephfs_7-node-cluster.yaml
```

Comment:
Can you also add the conf file to use at the beginning of the suite, which helps others when they try to run.